resources/lib/model.py

```diff
@@ -239,6 +239,23 @@ def __init__(self, data: dict):
             self.playcount = 1 if (int(self.playhead / self.duration * 100)) > 90 else 0
```

Comment:
We should put it to utils too since playhead is in panel

Reply:
Not sure what you mean by moving it to utils, but it's not on panel, but on the same level as panel.